<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorConfigW</class>
 <widget class="QDialog" name="MotorConfigW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="invertMotBol">
      <property name="text">
       <string>Invert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="driveMotBol">
      <property name="text">
       <string>Drive Motor</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="saveSetBtn">
      <property name="text">
       <string>Save Settings</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="MotorNamLbl">
      <property name="text">
       <string>Motor Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="motorNamEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>281</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor Config</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
